common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -35,6 +35,8 @@
 
 from openedx.core.djangolib.markup import HTML, Text
 
+from opaque_keys.edx.locator import CourseLocator
```

Comment:
I think we don't need this and can simply convert `self.course_id` to string by doing `unicode(self.course_id)` and then compare it with any course id string.